frontend/src/app/account/register.component.html

```diff
@@ -12,42 +15,89 @@ <h2 class="mt-0 align-self-start">Register</h2>
             }
         </div>
         <div class="form-field">
-            <label for="email">Email</label>
+            <div class="flex flex-row align-items-center gap-2">
+                <i class="pi pi-envelope"></i>
+                <label for="email">Email</label>
+            </div>
             <input id="email" type="email" formControlName="email" pInputText />
             @if (isEmailInvalid) {
                 <small class="text-red-300">The provided email is invalid</small>
             }
         </div>
         <div class="form-field">
-            <label for="password">Password</label>
+            <div class="flex flex-row align-items-center gap-2">
+                <i class="pi pi-lock"></i>
+                <label for="password">Password</label>
+            </div>
             <p-password
-                id="password"
+                inputId="password"
+                name="password"
                 class="p-fluid"
                 formControlName="password"
-                [strongRegex]="strongPasswordRegex"
-                [toggleMask]="true">
-                <ng-template pTemplate="footer">
-                    <p-divider />
-                    <p class="mt-1 text-sm">Your password must contain:</p>
-                    <ul class="pl-2 ml-2 mt-0 text-sm line-height-3">
-                        <li>At least one lowercase</li>
-                        <li>At least one uppercase</li>
-                        <li>At least one numeric</li>
-                        <li>At least one special character</li>
-                        <li>Minimum 8 characters</li>
-                    </ul>
-                </ng-template>
+                [toggleMask]="true"
+                [feedback]="false">
             </p-password>
             @if (isPasswordWeak) {
                 <small class="text-red-300">The provided password is too weak</small>
             } @else if (isPasswordInvalid) {
                 <small class="text-red-300">The provided password contains invalid characters</small>
             }
+            <ul class="pl-0 m-0 text-sm line-height-3">
+                <li class="flex gap-2 align-items-center">
+                    @if (isPasswordLowercase) {
```

Comment:
Could we also hide the criteria until the user starts modifying the password field (i.e. `passwordControl.dirty`), and hide the criteria once all are satisfied (`passwordControl.valid`)?

Reply:
Yup good idea. Was thinking whether I should do that haha